src/components/Form/form.ts

```diff
@@ -41,8 +43,10 @@ class Form extends Block<FormProps> {
         }
       }
     });
+
     return Object.values(children)
-        .every((input)=>input && input instanceof Input && input.props.isValid);
+        .filter((child)=>child instanceof Input)
+        .every((input)=>input && input.props.isValid);
```

Comment:
Этот момент тоже разнится по проекту
```suggestion
        .filter((child) => child instanceof Input)
        .every((input) => input && input.props.isValid);
```